views/account/settings/index.ejs

```diff
@@ -9,41 +9,144 @@
   <body class="h-100 text-center text-white bg-dark">
     <div class="row col-12"><%- include('../../partials/navbar'); %></div>
 
-    <div
-      class="cover-container w-100 d-flex align-items-center p-3 mx-auto"
-    >
-      <div class="row col-12">
-        <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
-        </main>
-      </div>
+    <!-- Parse the user's settings as JSON before using. -->
+    <% var settings = JSON.parse(user.settings); %>
+
+    <!-- Create the options. -->
+    <%var options = { font_size: ["12pt", "14pt", "16pt", "18pt"], font_family:
```

Comment:
These options are also not set in stone... if anyone has ideas feel free to share them with the group.

Reply:
The font size options should be chosen with consideration for how they look on the webpage. We should make sure the largest, 18pt font, wouldn't alter the page in any serious way

Reply:
Agreed. I also think more fonts may emerge once we start user testing, but the code is currently flexible enough to handle changing these.